source/renderer/app/stores/NodeUpdateStore.js

```diff
@@ -57,4 +62,21 @@ export default class NodeUpdateStore extends Store {
     this.isNotificationExpanded = !this.isNotificationExpanded;
   };
 
+  @action _getLatestAvailableAppVersion = async () => {
+    const { version, platform, isDevelopment } = this.environment;
+
+    if (!isDevelopment) {
+      this.isNewAppVersionLoading = true;
+      const versionInfo = await this.api.ada.getLatestAppVersionInfo();
+      const availableVersion = get(versionInfo, ['platforms', platform, 'version'], null);
+      const isNewAppVersionAvailable = availableVersion && availableVersion > version;
```

Comment:
@tomothespian are you sure this comparison works? Shouldn't we compare all 3 different segments of the version? E.g. individual major, middle and minor version checks?